react-basic-practice/src/components/Modal/index.tsx

```diff
@@ -39,27 +43,54 @@ const Modal: React.FC<ModalProps> = ({
         >
           {title}
         </ModalHeader>
-        <ModalBody padding="19px 0 0 0">{children}</ModalBody>
+        <ModalBody padding="19px 0 0 0">
+          {isConfirmDelete ? (
+            <Box textAlign="center">
+              <Text>Are you sure you want to delete this customer?</Text>
```

Comment:
Pls add props like `confirmDescription` for the text

Reply:
i done